distribution/docs/src/main/resources/DDFOmniContents.adoc

```diff
@@ -7,66 +75,92 @@ include::ManContents.adoc[]
 
 <<<
 
+== Managing {branding} Admin
+
 include::${project.build.directory}/apps/admin/ManContents.adoc[]
 
 <<<
 
+== Managing {branding} Catalog
+
 include::${project.build.directory}/apps/catalog/ManContents.adoc[]
 
 <<<
 
+== Managing {branding} Content
+
 include::${project.build.directory}/apps/content/ManContents.adoc[]
 
 <<<
 
+== Managing {branding} Platform
+
 include::${project.build.directory}/apps/platform/ManContents.adoc[]
 
 <<<
 
+== Managing {branding} Security
+
 include::${project.build.directory}/apps/security/ManContents.adoc[]
 
 <<<
 
+== Managing {branding} Spatial
+
 include::${project.build.directory}/apps/spatial/ManContents.adoc[]
 
 <<<
 
+== Managing {branding} UI
+
 include::${project.build.directory}/apps/ui/ManContents.adoc[]
 
 <<<
 
 == Integrating {branding}
 
-<<<
-
```

Comment:
Yes the ${project.build.directory} gets resolved by Maven filtering before the docs are rendered.
The {branding} gets rendered by asciidoctor, even in titles.
(the $ is significant for this reason)